src/main/java/uk/gov/hmcts/cmc/claimstore/models/offers/PartyStatement.java

```diff
@@ -7,16 +7,25 @@
 @JsonInclude(JsonInclude.Include.NON_ABSENT)
 public class PartyStatement {
 
-    private final StatementType type;
-    private final MadeBy madeBy;
-    private final Offer offer;
+    private StatementType type;
+    private MadeBy madeBy;
+    private Offer offer;
+
+    public PartyStatement() {
+        // default contractor required by Jackson
```

Comment:
slightly misleading comment, its only required because you have added a second constructor.
and that's required you to not make the parameters final.
I think it would be nicer to go back to the one constructor and allow Offer to be null.

Otherwise keep the two constructors and annotate the one with all the parameters to be used by jackson

Reply:
Beside what Tim, said, it says _default **contractor**_ 😉

Reply:
updated